cdr/CDR-001-council.md

```diff
@@ -0,0 +1,32 @@
+# CDR 001 - Council
+
+## Status
+
+Proposed
+
+## Context
+
+Rawkode Academy is currently run by [@rawkode](https://github.com/rawkode) with the help of some people from the Discord community.
+To make decisions regarding
+
+- Communication channels - Should we use Discord? Should we use Matrix? Use GitHub Discussion?
+- Events - Another Hackathon?
+- Content - Community blog?
+- ...
+
+easier, a group of people (which includes [@rawkode](https://github.com/rawkode)) should discuss said topics and put them to a vote with the community.
+
+## Decision
+
+A council of 3 (to get a quorum) meets once a month to discuss the aforementioned points / topics. The council session happens in public and each decision should be written down as an ADR.
```

Comment:
Let's go with 5 or 7. I don't want this to be a burden on anyone and if they're unavailable, decisions should still be able to proceed.

Reply:
Updated to 5. We start with 5 and see how it goes :)